api/PayrollProcessor.Web.Api/Features/Notifications/NotificationController.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace PayrollProcessor.Web.Api.Features.Notifications
 {
-    [Route("api/[controller]")]
+    [Route("notification")]
```

Comment:
The `api/` was doubling up `api` in the route path. We already apply `api/` globally in our `MvcOptionsRouteExtensions`.